app/models/concerns/digital_object_concerns/publish_behavior.rb

```diff
@@ -81,7 +88,10 @@ def perform_publish_changes(publish_to: [], unpublish_from: [], publishing_user:
             # Remove entries associated with successfull unpublishes AND successful publishes.
             # For successful publishes that need time/user updates, it's easier to do a mass deletion
             # than to find and update specific ones.
-            self.publish_entries.destroy_by(publish_target: successful_publish_targets + successful_unpublish_targets)
+            successful_additons_and_removals = successful_publish_targets + successful_unpublish_targets
```

Comment:
Just flagging a typo here (and in the reference below it): should be successful_additions_and_removals

Reply:
Ah!  Thanks for catching that!  Fixed!